components/bl-stopwatch/src/index.js

```diff
@@ -45,12 +45,12 @@ export default function Stopwatch({ component }) {
       const startTime = Date.now();
 
       timerRef.current = setInterval(() => {
-        const currentTime = new Date().getTime();
-        const elapsedTime = currentTime - startTime;
+        const currentTime = Date.now();
+        const elapsedTime = currentTime - startTime + time.elapsedTime;
```

Comment:
To me, there is no need of `tickRate` property and using Date.now() to get long value after dot for seconds. 
I prefer add one more setting which will show or hide milliseconds as a three-digit number. In this case we could simplify the timer logic.
But I guess this question is more to @snaqqqystas.